tasks/task12.ipynb

```diff
@@ -0,0 +1,265 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Soret effect\n",
+    "\n",
+    "This task will demonstrate how to setup a FESTIM model including the Soret effect.\n",
+    "\n",
+    "When including the Soret effect, the flux of hydrogen is not only driven by the gradient of concentration:\n",
+    "$$\n",
+    "J = -D \\nabla c_\\mathrm{m} - D\\frac{Q^* c_\\mathrm{m}}{k_B T^2} \\nabla T\n",
+    "$$\n",
+    "\n",
+    "In other words, the hydrogen flux goes from hot regions to cold regions.\n",
```

Comment:
I suppose it depends on the sign of $Q^*$

Reply:
That's true, I'll remove this

Reply:
The rest is fine for me.